### PR TITLE
[DOCS] Added contributing section to the documentation

### PR DESCRIPTION
**Pull request by MagdaDziadosz**
Added contributing section to the documentation:

- [x] Overview 
    - [x] Tracking issues - included in overview
-  [x] Workflow
    - [x] Fork the project
    - [x] Commit and push the change
    - [x] Open a pull request
- [x] Coding guidelines - waiting for dev input
- [x] Testing
    - [x] Contributing to documentation - included in workflow and testing

Jira ticket: [HEL-27](https://mirantis.jira.com/browse/HEL-27)

Add the Contributing section

### CODE REVIEW

File: docs/contributing/contribute-testing.md
Comment (makhov) [inline docs/contributing/contribute-testing.md:49]: This paragraph is redundant for k0smotron

File: docs/contributing/contribute-testing.md
Comment (makhov) [inline docs/contributing/contribute-testing.md:59]: This section is also redundant

File: README.md
Comment (KoryKessel-Mirantis) [inline README.md:87]: I suggest flipping the clauses in the second sentence to read:

To learn how you can contribute to the growth and enhancement of k0smotron, please read our [Contribution Guidelines](docs/contributing/contribute-overview.md).

**Review by makhov — APPROVED** [review]

File: docs/contributing/contribute-testing.md
Comment (KoryKessel-Mirantis) [inline docs/contributing/contribute-testing.md:71]: Suggest:  To clean up the local workspace, run `make clean'. This removes all the intermediate files and directories that were created during the k0smotron build.

File: docs/contributing/contribute-workflow.md
Comment (KoryKessel-Mirantis) [inline docs/contributing/contribute-workflow.md:4]: Create pull requests to...

File: docs/contributing/contribute-workflow.md
Comment (KoryKessel-Mirantis) [inline docs/contributing/contribute-workflow.md:10]: Suggest "In the top right-hand corner, "

File: docs/contributing/contribute-workflow.md
Comment (KoryKessel-Mirantis) [inline docs/contributing/contribute-workflow.md:49]: Suggest: "Create and switch to a..."

File: docs/contributing/contribute-workflow.md
Comment (KoryKessel-Mirantis) [inline docs/contributing/contribute-workflow.md:98]: Suggest: "To add..."

File: docs/contributing/contribute-workflow.md
Comment (KoryKessel-Mirantis) [inline docs/contributing/contribute-workflow.md:100]: Suggest: Push your changes to the fork repository.

File: docs/contributing/contribute-workflow.md
Comment (KoryKessel-Mirantis) [inline docs/contributing/contribute-workflow.md:108]: Suggest: "Refer to the official GitHub documentation [Creating a pull request from a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).

File: docs/contributing/contribute-workflow.md
Comment (KoryKessel-Mirantis) [inline docs/contributing/contribute-workflow.md:112]: Suggest: after 'pull request' insert 'and leave comments.'

File: docs/contributing/contribute-workflow.md
Comment (KoryKessel-Mirantis) [inline docs/contributing/contribute-workflow.md:115]: Need commas.

File: docs/contributing/contribute-workflow.md
Comment (KoryKessel-Mirantis) [inline docs/contributing/contribute-workflow.md:121]: Separate 'Squash' sentence out to a new paragraph and attach the sentence following to it.

File: docs/contributing/contribute-workflow.md
Comment (KoryKessel-Mirantis) [inline docs/contributing/contribute-workflow.md:133]: If necessary, for whatever reason...

**Review by KoryKessel-Mirantis — CHANGES_REQUESTED** [review]

**Comment by juanluisvaladas** [comment]
LGTM both as is or with the suggestions work for me as well.
Needs rebase, feel free to ping me on slack for stamping it.

File: docs/contributing/contribute-overview.md
Comment (KoryKessel-Mirantis) [reply docs/contributing/contribute-overview.md]: LGTM

File: docs/contributing/contribute-overview.md
Comment (KoryKessel-Mirantis) [reply docs/contributing/contribute-overview.md]: LGTM

File: docs/contributing/contribute-overview.md
Comment (KoryKessel-Mirantis) [reply docs/contributing/contribute-overview.md]: LGTM

File: docs/contributing/contribute-overview.md
Comment (KoryKessel-Mirantis) [reply docs/contributing/contribute-overview.md]: LGTM

File: docs/contributing/contribute-overview.md
Comment (KoryKessel-Mirantis) [reply docs/contributing/contribute-overview.md]: LGTM

File: docs/contributing/contribute-overview.md
Comment (KoryKessel-Mirantis) [reply docs/contributing/contribute-overview.md]: 'followed' should be 'follows'

File: docs/contributing/contribute-testing.md
Comment (KoryKessel-Mirantis) [inline docs/contributing/contribute-testing.md:61]: I suggest rethinking this edit.

First, can a PR be merged if any of these conditions are not fulfilled? The 'should' raises questions where perhaps questions should not be raised.

Second, if it is not a 'should' but is indeed a 'must', then the conditions should be presented in the past tense, as they were prior to this edit.

If the conditions are a 'must', I suggest:

For a pull request to be merged, all of the following must be true:

1. The PR has passed all of the automated tests (style, build and conformance tests).
2. All of the PR commits have been signed with the `--signoff` option.
3. The PR has been reviewed and approved by a code owner.
4. The PR has been rebased against upstream main branch.

File: docs/contributing/contribute-testing.md
Comment (KoryKessel-Mirantis) [inline docs/contributing/contribute-testing.md:66]: Suggest:

Run 'make clean' to remove all the intermediate files and directories created locally during the k0smotron build.

**Review by KoryKessel-Mirantis — CHANGES_REQUESTED** [review]

**Comment by MagdaDziadosz** [comment]
@juanluisvaladas ready for final check

**Review by KoryKessel-Mirantis — APPROVED** [review]

**Pull request merged**
